### PR TITLE
python3Packages.glasgow: unstable-2020-02-08 -> unstable-2020-06-29

### PR DESCRIPTION
**Pull request by matthuszagh**
###### Motivation for this change
Update glasgow to an unstable version from 06-29. This adds the analyze applet.

###### Things done
I've tried using this and it has worked as expected.

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by flokli — CHANGES_REQUESTED** [review]
Thanks for the PR! Please name the commit messages according to CONTRIBUTING.md.

**Comment by matthuszagh** [comment]
@flokli thanks for pointing out the mistake. It should now be correct. Let me know if there are any other issues.

File: pkgs/development/python-modules/fx2/default.nix [hunk 1]
```diff
@@ -7,15 +7,15 @@
 , crcmod
 }:
 
-buildPythonPackage {
+buildPythonPackage rec {
   pname = "fx2";
-  version = "unstable-2020-01-25";
+  version = "v0.9";
```
Comment (flokli) [inline pkgs/development/python-modules/fx2/default.nix:12]:
```suggestion
  version = "0.9";
```

**Review by flokli — CHANGES_REQUESTED** [review]
The version string shouldn't include a `v`. Apart from that, built and looks good.

**Comment by matthuszagh** [comment]
Done. I removed the `v` from the commit message too.

**Pull request merged**